tests/simulator/py/fluid_init.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+#
+# formatted with black
+
+from tests.simulator.py import InitValueValidation
+
+import phare.pharein as ph, numpy as np
+from phare.pp.diagnostics import Diagnostics
+
+
+class FluidInitValidation(InitValueValidation):
+
+    pop_diag_file = lambda fn_name: "ions_pop_ions_${POP}_" + fn_name
+
+    def test_1d(self):
+        from phare.pp.diagnostics import _Fluid
+        from tests.diagnostic import dump_all_diags
+
+        diag_out_dir = "phare_outputs/fluid_init"
+        dic = InitValueValidation.diag_options(diag_out_dir)
+        dic.update(
+            {
+                "diags_fn": lambda model: dump_all_diags(model.populations),
+                "populations": {
+                    "beta": {"nbr_part_per_cell": 10000, "init": {"seed": 133337}},
+                    "gamma": {"nbr_part_per_cell": 100000, "init": {"seed": 1333337}},
+                },
+            }
+        )
+
+        diags = self._simulate_diagnostics(dim=1, interp=1, input=dic)[_Fluid.__name__]
+        self._checkFluidFromDensityFN(diags)
+        self._checkFluidFromFluxFN(diags)
+        self._checkIons(diags)
+
+    def _sum_pop_patches(self, pop_patches, dataset):
+        return np.sum(
+            [
+                self._aggregate_physical_patch_value(patches, True)[dataset]
+                for pop, patches in pop_patches.items()
+            ],
+            axis=0,
+        )
+
+    def _checkIons(self, fluids):
```

Comment:
I think the argument `fluids` should be `fluid_diags` so we know they are diagnostics objects